code/addons/themes/docs/getting-started/postcss.md

```diff
@@ -0,0 +1,116 @@
+# 🏁 Getting started with `postcss`
+
+## 📦 Install addon
+
+<!-- **NOTE:** As of Storybook 7.2, `@storybook/addon-themes` ships in `@storybook/addon-essentials`. If you're using Storybook >= 7.2, skip to ["Import your css"](#🥾-import-your-css). -->
```

Comment:
I think that we can just remove this line 😁

Reply:
Done https://github.com/ai/storybook/commit/69d414f758f991d731f84c30412e9628b79d90f0